search/typesenseGoandJS/README.md

```diff
@@ -0,0 +1,14 @@
+INSTALL TYPESENSE VIA DOCKER:
+docker pull typesense/typesense:0.22.2
+
+MAKE DIRECTORY:
+mkdir /tmp/typesense-data
+
+START UP THE TYPESENSE SERVER:
+docker run -p 8108:8108 -v/tmp/typesense-data:/data typesense/typesense:0.22.2 \--data-dir /data --api-key=xyz --enable-cors
+
+index files:
+go run index.go
+
```

Comment:
running go run index.go triggered
go: github.com/typesense/typesense-go@v0.4.0 requires
        github.com/golang/mock@v1.6.0: missing go.sum entry; to add it:
        go mod download github.com/golang/mock
make sure everythng is added to go.sum

Reply:
Running index.go triggered error:
panic: status: 404 response: {"message": "No collection with name `featuredata` found."}

I fixed it by commenting out lines 53-56